theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/AggregatedKeyframeTrack/AggregatedKeyframeTrack.tsx

```diff
@@ -79,7 +78,7 @@ function AggregatedKeyframeTrack_memo(props: IAggregatedKeyframeTracksProps) {
       layoutP={layoutP}
       viewModel={viewModel}
       aggregateKeyframes={posKfs}
-      key={'agg-' + position}
+      key={'agg-' + keyframes[0].kf.id}
```

Comment:
@AndrewPrifer , can you share additional info on what the issue was, here? Is this a perf optimization, for when the keyframes are being dragged, it doesn't cause a complete re-render?